test/client.js

```diff
@@ -348,6 +443,64 @@ function makeRequestAndExpectUrl (client, i, t, cb) {
   })
 }
 
+test('20 times HEAD with pipelining 10', (t) => {
```

Comment:
This test fails and I still can't figure out why. If I run manually a client with pipelining = 10 I have no issues.

For example:
```js
const { Client } = require('/path/to/undici')

const client = new Client('http://localhost:3000')

for (let i = 0; i < 100; i++) {
    client.request({
        path: '/',
        method: 'HEAD',
        pipelining: 10
    }, (error, { statusCode, body }) => {
        console.error(error)
        console.log(statusCode)
    })
}
```
works fine.

Reply:
I used the server in `benchmarks/origins.js` to test.

Reply:
this should be resolved now?

Reply:
this should be resolved now?